src/app/suggestion-notifications/reciter-suggestions/suggestions-popup/suggestions-popup.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { SuggestionTargetsStateService } from '../suggestion-targets/suggestion-targets.state.service';
+import { NotificationsService } from '../../../shared/notifications/notifications.service';
+import { SuggestionsService } from '../suggestions.service';
+import { takeUntil } from 'rxjs/operators';
+import { SuggestionTarget } from '../../../core/suggestion-notifications/reciter-suggestions/models/suggestion-target.model';
+import { isNotEmpty } from '../../../shared/empty.util';
+import { combineLatest, Subject } from 'rxjs';
+
+@Component({
+  selector: 'ds-suggestions-popup',
+  templateUrl: './suggestions-popup.component.html',
+  styleUrls: ['./suggestions-popup.component.scss']
+})
+export class SuggestionsPopupComponent implements OnInit, OnDestroy {
+
+  labelPrefix = 'mydspace.';
```

Comment:
Is this component only used on mydspace?  If not, then it may need a different label prefix.  It could use the same prefix as the keys on the Publication Claim page.